task/07-yield-tasks.js

```diff
@@ -33,7 +33,21 @@
  *
  */
 function* get99BottlesOfBeer() {
-    throw new Error('Not implemented');
+    let i = 99;
+    let flag = true;
+    while (i > 0 || !flag) {
+        let message = i === 0 ? "no more bottles" : i === 1 ? "1 bottle" : `${i} bottles`;
```

Comment:
Please, avoid this code style, if-else is more readable than multiple conditional operators ? :